tool/ciphers.cc

```diff
@@ -21,19 +21,69 @@
 #include <openssl/ssl.h>
 
 #include "internal.h"
+#include "../ssl/internal.h"
 
+static const argument_t kArguments[] = {
+    {
+        "-openssl-name", kBooleanArgument,
+        "Print OpenSSL-style cipher names instead of IETF cipher names.",
+    },
```

Comment:
Instead of a boolean should this be a enum? Is there ever a potential 3rd type of value to print? Such as the id value from the relevant RFC, or printing out a combined map?

Reply:
Nevermind, I see this is just a refactor and it was already just a boolean flag. Something to worry about in the future.